### PR TITLE
fittrackee: v0.9.3 -> v0.9.4

### PR DESCRIPTION
**Pull request by tebriel**
Fittrackee Changelog: https://github.com/SamR1/FitTrackee/releases/tag/v0.9.4

required bumping python humanize

Python Humanize Changelog: https://github.com/python-humanize/humanize/releases/tag/4.12.2

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by tebriel** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 395755`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 18 packages marked as broken and skipped:</summary>
  <ul>
    <li>opsdroid</li>
    <li>opsdroid.dist</li>
    <li>python312Packages.elegy</li>
    <li>python312Packages.elegy.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>python312Packages.treex</li>
    <li>python312Packages.treex.dist</li>
    <li>python313Packages.cassandra-driver</li>
    <li>python313Packages.cassandra-driver.dist</li>
    <li>python313Packages.swh-export</li>
    <li>python313Packages.swh-export.dist</li>
    <li>python313Packages.swh-objstorage</li>
    <li>python313Packages.swh-objstorage.dist</li>
    <li>python313Packages.swh-scheduler</li>
    <li>python313Packages.swh-scheduler.dist</li>
    <li>python313Packages.swh-storage</li>
    <li>python313Packages.swh-storage.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 32 packages failed to build:</summary>
  <ul>
    <li>python312Packages.awswrangler</li>
    <li>python312Packages.awswrangler.dist</li>
    <li>python312Packages.bambi</li>
    <li>python312Packages.bambi.dist</li>
    <li>python312Packages.nutpie</li>
    <li>python312Packages.nutpie.dist</li>
    <li>python312Packages.pymc</li>
    <li>python312Packages.pymc.dist</li>
    <li>python313Packages.awswrangler</li>
    <li>python313Packages.awswrangler.dist</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-checkpoint.dist</li>
    <li>sourcehut.buildsrht</li>
    <li>sourcehut.buildsrht.dist</li>
    <li>sourcehut.gitsrht</li>
    <li>sourcehut.gitsrht.dist</li>
    <li>sourcehut.hgsrht</li>
    <li>sourcehut.hgsrht.dist</li>
    <li>sourcehut.hubsrht</li>
    <li>sourcehut.hubsrht.dist</li>
    <li>sourcehut.listssrht</li>
    <li>sourcehut.listssrht.dist</li>
    <li>sourcehut.mansrht</li>
    <li>sourcehut.mansrht.dist</li>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.metasrht.dist</li>
    <li>sourcehut.pastesrht</li>
    <li>sourcehut.pastesrht.dist</li>
    <li>sourcehut.todosrht</li>
    <li>sourcehut.todosrht.dist</li>
    <li>wapiti</li>
    <li>wapiti.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 173 packages built:</summary>
  <ul>
    <li>authentik</li>
    <li>cassandra (cassandra_4)</li>
    <li>cassandra_3_0</li>
    <li>cassandra_3_11</li>
    <li>charmcraft</li>
    <li>charmcraft.dist</li>
    <li>datalad (python312Packages.datalad)</li>
    <li>datalad-gooey (python312Packages.datalad-gooey)</li>
    <li>datalad-gooey.dist (python312Packages.datalad-gooey.dist)</li>
    <li>datalad.dist (python312Packages.datalad.dist)</li>
    <li>datatrove</li>
    <li>datatrove.dist</li>
    <li>devtoolbox</li>
    <li>fittrackee</li>
    <li>fittrackee.dist</li>
    <li>ghunt</li>
    <li>ghunt.dist</li>
    <li>gnome-feeds</li>
    <li>gnome-keysign</li>
    <li>gnome-keysign.dist</li>
    <li>klaus (python312Packages.klaus)</li>
    <li>klaus.dist (python312Packages.klaus.dist)</li>
    <li>magic-wormhole (python312Packages.magic-wormhole)</li>
    <li>magic-wormhole.dist (python312Packages.magic-wormhole.dist)</li>
    <li>marge-bot</li>
    <li>marge-bot.dist</li>
    <li>matrix-synapse-plugins.matrix-synapse-s3-storage-provider</li>
    <li>matrix-synapse-plugins.matrix-synapse-s3-storage-provider.dist</li>
    <li>minari (python312Packages.minari)</li>
    <li>minari.dist (python312Packages.minari.dist)</li>
    <li>mktxp</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>paperless-ngx</li>
    <li>pg_activity</li>
    <li>pg_activity.dist</li>
    <li>prefect</li>
    <li>prefect.dist</li>
    <li>python312Packages.ale-py</li>
    <li>python312Packages.ale-py.dist</li>
    <li>python312Packages.arviz</li>
    <li>python312Packages.arviz.dist</li>
    <li>python312Packages.cassandra-driver</li>
    <li>python312Packages.cassandra-driver.dist</li>
    <li>python312Packages.corner</li>
    <li>python312Packages.corner.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.datalad-next</li>
    <li>python312Packages.datalad-next.dist</li>
    <li>python312Packages.delorean</li>
    <li>python312Packages.delorean.dist</li>
    <li>python312Packages.evosax</li>
    <li>python312Packages.evosax.dist</li>
    <li>python312Packages.flax</li>
    <li>python312Packages.flax.dist</li>
    <li>python312Packages.flower</li>
    <li>python312Packages.flower.dist</li>
    <li>python312Packages.flowjax</li>
    <li>python312Packages.flowjax.dist</li>
    <li>python312Packages.gehomesdk</li>
    <li>python312Packages.gehomesdk.dist</li>
    <li>python312Packages.gekitchen</li>
    <li>python312Packages.gekitchen.dist</li>
    <li>python312Packages.gremlinpython</li>
    <li>python312Packages.gremlinpython.dist</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.heudiconv</li>
    <li>python312Packages.heudiconv.dist</li>
    <li>python312Packages.humanize</li>
    <li>python312Packages.humanize.dist</li>
    <li>python312Packages.jinja2-humanize-extension</li>
    <li>python312Packages.jinja2-humanize-extension.dist</li>
    <li>python312Packages.karton-mwdb-reporter</li>
    <li>python312Packages.karton-mwdb-reporter.dist</li>
    <li>python312Packages.maya</li>
    <li>python312Packages.maya.dist</li>
    <li>python312Packages.mqtt2influxdb</li>
    <li>python312Packages.mqtt2influxdb.dist</li>
    <li>python312Packages.mwdblib</li>
    <li>python312Packages.mwdblib.dist</li>
    <li>python312Packages.numpyro</li>
    <li>python312Packages.numpyro.dist</li>
    <li>python312Packages.orbax-checkpoint</li>
    <li>python312Packages.orbax-checkpoint.dist</li>
    <li>python312Packages.pettingzoo</li>
    <li>python312Packages.pettingzoo.dist</li>
    <li>python312Packages.pins</li>
    <li>python312Packages.pins.dist</li>
    <li>python312Packages.pycron</li>
    <li>python312Packages.pycron.dist</li>
    <li>python312Packages.radish-bdd</li>
    <li>python312Packages.radish-bdd.dist</li>
    <li>python312Packages.secure</li>
    <li>python312Packages.secure.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.stable-baselines3</li>
    <li>python312Packages.stable-baselines3.dist</li>
    <li>python312Packages.swh-export</li>
    <li>python312Packages.swh-export.dist</li>
    <li>python312Packages.swh-objstorage</li>
    <li>python312Packages.swh-objstorage.dist</li>
    <li>python312Packages.swh-scheduler</li>
    <li>python312Packages.swh-scheduler.dist</li>
    <li>python312Packages.swh-storage</li>
    <li>python312Packages.swh-storage.dist</li>
    <li>python312Packages.tianshou</li>
    <li>python312Packages.tianshou.dist</li>
    <li>python312Packages.torchio</li>
    <li>python312Packages.torchio.dist</li>
    <li>python312Packages.vqgan-jax</li>
    <li>python312Packages.vqgan-jax.dist</li>
    <li>python312Packages.waymax</li>
    <li>python312Packages.waymax.dist</li>
    <li>python313Packages.datalad</li>
    <li>python313Packages.datalad-gooey</li>
    <li>python313Packages.datalad-gooey.dist</li>
    <li>python313Packages.datalad-next</li>
    <li>python313Packages.datalad-next.dist</li>
    <li>python313Packages.datalad.dist</li>
    <li>python313Packages.delorean</li>
    <li>python313Packages.delorean.dist</li>
    <li>python313Packages.flower</li>
    <li>python313Packages.flower.dist</li>
    <li>python313Packages.gehomesdk</li>
    <li>python313Packages.gehomesdk.dist</li>
    <li>python313Packages.gekitchen</li>
    <li>python313Packages.gekitchen.dist</li>
    <li>python313Packages.gremlinpython</li>
    <li>python313Packages.gremlinpython.dist</li>
    <li>python313Packages.humanize</li>
    <li>python313Packages.humanize.dist</li>
    <li>python313Packages.jinja2-humanize-extension</li>
    <li>python313Packages.jinja2-humanize-extension.dist</li>
    <li>python313Packages.karton-mwdb-reporter</li>
    <li>python313Packages.karton-mwdb-reporter.dist</li>
    <li>python313Packages.klaus</li>
    <li>python313Packages.klaus.dist</li>
    <li>python313Packages.magic-wormhole</li>
    <li>python313Packages.magic-wormhole.dist</li>
    <li>python313Packages.maya</li>
    <li>python313Packages.maya.dist</li>
    <li>python313Packages.mqtt2influxdb</li>
    <li>python313Packages.mqtt2influxdb.dist</li>
    <li>python313Packages.mwdblib</li>
    <li>python313Packages.mwdblib.dist</li>
    <li>python313Packages.pins</li>
    <li>python313Packages.pins.dist</li>
    <li>python313Packages.pycron</li>
    <li>python313Packages.pycron.dist</li>
    <li>python313Packages.radish-bdd</li>
    <li>python313Packages.radish-bdd.dist</li>
    <li>python313Packages.secure</li>
    <li>python313Packages.secure.dist</li>
    <li>python313Packages.torchio</li>
    <li>python313Packages.torchio.dist</li>
    <li>sourcehut.coresrht</li>
    <li>sourcehut.coresrht.dist</li>
    <li>swh</li>
    <li>tahoe-lafs</li>
    <li>tahoe-lafs.dist</li>
    <li>tahoe-lafs.doc</li>
    <li>tahoe-lafs.info</li>
    <li>terraform-compliance</li>
    <li>terraform-compliance.dist</li>
    <li>todoman</li>
    <li>todoman.dist</li>
    <li>twtxt</li>
    <li>twtxt.dist</li>
  </ul>
</details>

**Comment by pbsds** [comment]
All failures exist on hydra except python313Packages.orbax-checkpoint, which builds on my machine
